### ISSUE TITLE
Can't download CLOC cross validation set

### ISSUE THREAD

**Issue by HynDuf** [Yasir-Ghunaim/RealtimeOCL#4]
Hi, I'm an enthusiast wanting to run the experiments. However, I'm stuck at Step 3: Download CLOC cross-validation set with a 'permission denied' error. Is this intended? If I'm not allowed to download it, can I still run the experiments without the cross-validation set?

Thanks a lot.

**Comment by Yasir-Ghunaim** [comment]
Hi @HynDuf,
Sorry for the late reply.

Thanks for reporting this. It looks like the Google Cloud bucket hosting the cross-validation set has expired. We will try to resolve it as soon as possible. I will also try to send you a temporary link with the cross-validation set.

In the meantime, please note that you can still run experiments without the cross-validation. We mainly used it for selecting learning rates and other hyperparameters.

Regards,
Yasir

**Comment by HynDuf** [comment]
Thanks a lot. Just take your time. I successfully ran the experiments without the cross-validation set by manually turning off the flag "--valdiation" (it was set to True by default, and add the flag still set it to True). Btw, really great paper.

**Comment by Yasir-Ghunaim** [comment]
@@HynDuf, thanks for your encouraging words. We mainly use the flag --valdiation when measuring the FLOPs of a particular method. 

Here is a temporary link containing the validation set:

https://drive.google.com/file/d/1pu8fHz9uCp8z5Ec6sScO3L4y943OZ-il/view?usp=drivesdk
